timeseries/tests/conftest.py

```diff
@@ -31,3 +31,30 @@ def pytest_collection_modifyitems(config, items):
 def temp_model_path(tmp_path_factory):
     fn = tmp_path_factory.mktemp(str(uuid4())[:6])
     return str(fn)
+
+
+@pytest.fixture(scope="session")
+def hf_model_path(tmp_path_factory):
+    """Force HuggingFace to cache the model config once and reuse it from a temporary cache directory.
+    This prevents inflating HuggingFace download numbers as an HTTP request is sent every time
+    ``ChronosPipeline.from_pretrained`` is called.
+    """
+    model_hub_id = "amazon/chronos-t5-tiny"
+    # get cache path for huggingface model
+    cache_path = tmp_path_factory.mktemp("hf_hub_cache")
+
+    try:
+        from autogluon.timeseries.models.chronos.pipeline import ChronosPipeline
+
+        # download and cache model from hf hub
+        _ = ChronosPipeline.from_pretrained(model_hub_id, cache_dir=str(cache_path))
+
+        # get model snapshot path
+        snapshots_path = cache_path / "models--amazon--chronos-t5-tiny" / "snapshots"
+        assert snapshots_path.exists()
+        snapshot_dir = next(snapshots_path.iterdir())
```

Comment:
Can there be multiple directories in `snapshots`?

Reply:
You can actually:

```
(base) ➜  snapshots ll /home/ubuntu/.cache/huggingface/hub/models--amazon--chronos-t5-base/snapshots
total 8.0K
drwxrwxr-x 2 ubuntu ubuntu 4.0K Mar 27 19:18 45d3b2421f623c7239897e753976bc883cb522d0
drwxrwxr-x 2 ubuntu ubuntu 4.0K Mar 20 17:03 c735c3bee35757ba75c4f11f0153b9838ad338de
```
Maybe read the commit hash for the main ref and use that?
```
(base) ➜  refs cat /home/ubuntu/.cache/huggingface/hub/models--amazon--chronos-t5-base/refs/main 
c735c3bee35757ba75c4f11f0153b9838ad338de%
```

Reply:
But maybe in the scope of CI you would only have one `main` ref and hash, so the current code should be okay.